src/screens/Customers/customerUtils.ts

```diff
@@ -0,0 +1,34 @@
+
+import { CustomerRecord, MeterReadingRecord } from '../../lib/airtable/interface';
+import { getPeriodFromDate, getCurrentPeriod, getLastPeriod } from '../../lib/moment/momentUtils';
+
+const isCurrentReading = (mr: MeterReadingRecord): boolean => {
+    const period: string = getPeriodFromDate(mr.date);
+    const currPer: string = getCurrentPeriod();
+    return period === currPer;
+};
+
+//Returns last reading in the period
+export const getCurrentReading = (customer: CustomerRecord): MeterReadingRecord | undefined => {
+  return customer.meterReadings.find(isCurrentReading);
+};
+
+const isStartingReading = (mr: MeterReadingRecord): boolean => {
+  const period: string = getPeriodFromDate(mr.date);
+  const lastPer: string = getLastPeriod();
+  return period === lastPer;
+};
```

Comment:
Doesn't this assume that the startingReading was from last period? I think we should have a broader definition of start reading: "The latest meter reading NOT from the current period". We should handle the case where a User is a little late to their readings or forgets a month (so there's no reading for the last period). 

This might require restructuring some of the functions